src/aks-preview/azext_aks_preview/agentpool_decorator.py

```diff
@@ -16,7 +16,10 @@
     AKSAgentPoolParamDict,
     AKSAgentPoolUpdateDecorator,
 )
-from azure.cli.core.azclierror import InvalidArgumentValueError
+from azure.cli.core.azclierror import (
```

Comment:
Sorry for the confusion, the `agentpool_decorator.py` is v2 decorator and currently not used. You could ignore this file and its corresponding test file `test_agentpool_decorator.py`.

Reply:
Removed the changes in ```agentpool_decorator``` and ```test_agentpool_decorator.py```